tap_iterable/iterable.py

```diff
@@ -108,9 +109,12 @@ def list_users(self, column_name=None, bookmark=None):
 
 
   def campaigns(self, column_name=None, bookmark=None):
+    bookmark = utils.strptime_with_tz(bookmark)
     res = self.get("campaigns")
     for c in res["campaigns"]:
```

Comment:
try to not use single letter variables, it reduces readability of the code
```suggestion
    for c in res["campaigns"]:
```

Reply:
This will be the part of refactoring code and this branch is specific to functionality changes. And, the line you have mentioned, I haven't made any changes to it.If I change this now, I have to be consistent about it for all the streams mentioned in the iterable.py.
Created the backlog ticket [TDL-22279](https://jira.talendforge.org/browse/TDL-22279) for the same and we can take care of it as part of later release as refactoring the code is not priority for the beta release.